src/tokenizer.rs

```diff
@@ -1876,11 +1900,29 @@ mod tests {
         compare(expected, tokens);
     }
 
+    #[test]
+    fn tokenize_quoted_identifier_with_no_escape() {
+        let sql = r#" "a "" b" "a """ "c """"" "#;
+        let dialect = GenericDialect {};
+        let mut tokenizer = Tokenizer::new(&dialect, sql, &TokenizerOptions { no_escape: true });
```

Comment:
```suggestion
        let opts = TokenizerOptions::new(&dialect).with_no_escape(true);
        let mut tokenizer = Tokenizer::new(opts, sql, &TokenizerOptions { no_escape: true });
```

Reply:
> [29cb1e2](https://github.com/sqlparser-rs/sqlparser-rs/pull/870/commits/29cb1e20f618b45ea9e9d9418723f8800bcaef8d)

I think this commit mentioned above captures this point. ;)